plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPlugin.groovy

```diff
@@ -32,6 +32,10 @@ public class AndroidCommandPlugin implements Plugin<Project> {
         defaultTask (project, 'disableSystemAnimations', 'disables system animations on the connected device', SystemAnimations) {
             enable = false
         }
+
+        project.tasks.withType(AdbTask) { task ->
+            extension.attachDefaults(task)
+        }
```

Comment:
I was thinking if it's possible to do it in 1 line, but I didn't figure it out when I was writing it. I think 3 lines is clear tho. What do you think?